application/base_script.sh

```diff
@@ -0,0 +1,40 @@
+xcp_d \
+    --participant_label 01
+    -t rest \
+    --bids-filter-file filter.json \
+    --combineruns \
```

Comment:
This only combined runs when X is matching in task-X, right? Not all runs?

Reply:
Right. It concatenates across runs where all other entities (including task) match. This means that mismatches on other entities (e.g., `dir`) won't be concatenated.